tests/sqlite3/test-sqlite3.cpp

```diff
@@ -413,6 +413,233 @@ TEST_CASE("SQLite long long", "[sqlite][longlong]")
     CHECK(v2 == v1);
 }
 
+// Test the DDL and metadata functionality
+TEST_CASE("SQLite DDL with metadata", "[sqlite][ddl]")
+{
+    if (sqlite3_libversion_number() < 3036000) {
```

Comment:
I don't know why this doesn't compile in the CI builds, maybe `sqlite.h` is not included from here?

We could wrap this function at the backend level or just do `select sqlite_version();` here perhaps?

Reply:
I think there is a namespace wrapper in the inclusion of the sqlite header, I will just add the namespace and do your proposed change.

Reply:
I tested the compil and test on my side, it is working as intended.